langchain/src/tools/index.ts

```diff
@@ -2,6 +2,7 @@ export { SerpAPI, type SerpAPIParameters } from "./serpapi.js";
 export { DadJokeAPI } from "./dadjokeapi.js";
 export { BingSerpAPI } from "./bingserpapi.js";
 export { Tool, type ToolParams, StructuredTool } from "./base.js";
+export { ConneryService } from "./connery.js";
```

Comment:
Remove and then you'll need to create new entrypoints for the new tool + toolkit to get exports working properly:

https://github.com/langchain-ai/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md